cflib/crazyflie/param.py

```diff
@@ -358,6 +359,36 @@ def get_value(self, complete_name, timeout=60):
         [group, name] = complete_name.split('.')
         return self.values[group][name]
 
+    def get_default_value(self, complete_name, callback=None):
+        """
+        Clear the current value of the specified parameter. from
```

Comment:
The description seems to be copy/pasted from the clear function :-)

Reply:
Yes :) Some kind of frankenstein state, have fixed with force push